src/metrics/listener.ts

```diff
@@ -124,8 +127,13 @@ export class MetricsListener {
       } catch (error) {
         logError("couldn't decrypt kms api key", { innerError: error });
       }
-    } else {
-      logError("api key not configured");
+    } else if (!config.logForwarding) {
+      const errorMessage = "api key not configured, see https://dtdg.co/sls-node-metrics";
+      if (config.logForwarding) {
+        logDebug(errorMessage);
```

Comment:
It doesn't appear that this will ever execute because the `if` condition is the opposite of the condition above it.

Reply:
Good catch @brianhyder